src/telemetry/step_stats.rs

```diff
@@ -35,6 +35,7 @@ impl CsvExporter for Metrics {
         // then dump them to the provided Write interface
         // TODO: include role dimension. That requires rethinking `Metrics` implementation
         // because it does not allow such breakdown atm.
+        #[cfg(not(feature = "step-trace"))]
```

Comment:
maybe we can contain this feature flag to be inside the benchmark only, by configuring test world and metrics by optiionally taking an argument not to print the header

Reply:
done!